third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -221,7 +221,7 @@ region are guaranteed to support the same version.
     Automatically send metrics from pods in the cluster to the Google Cloud Monitoring API.
     VM metrics will be collected by Google Compute Engine regardless of this setting
     Available options include
-    `monitoring.googleapis.com`, `monitoring.googleapis.com/kubernetes`, and `none`.
+    `monitoring.googleapis.com`(`Legacy Stackdriver`), `monitoring.googleapis.com/kubernetes`(`Stackdriver Kubernetes Engine Monitoring`), and `none`.
```

Comment:
```suggestion
    `monitoring.googleapis.com` (Legacy Stackdriver), `monitoring.googleapis.com/kubernetes` (Stackdriver Kubernetes Engine Monitoring), and `none`.
```

I think this is more clear if the names of the logging endpoints are unquoted

Reply:
updated